guides/source/action_text_overview.md

```diff
@@ -8,149 +8,145 @@ rich text content.
 
-Most WYSIWYG editors are wrappers around HTML’s `contenteditable` and `execCommand` APIs,
-designed by Microsoft to support live editing of web pages in Internet Explorer 5.5,
-and [eventually reverse-engineered](https://blog.whatwg.org/the-road-to-html-5-contenteditable#history)
-and copied by other browsers.
+INFO: Most WYSIWYG editors are wrappers around HTML’s `contenteditable` and `execCommand` APIs. These APIs were designed by Microsoft to support live editing of web pages in Internet Explorer 5.5. They were eventually reverse-engineered and copied by other browsers. Consequently, these APIs were never fully specified or documented, and because WYSIWYG HTML editors are enormous in scope, each browser's implementation has its own set of bugs and quirks. Hence, JavaScript developers are often left to resolve the inconsistencies.<br><br> Trix sidesteps these inconsistencies by treating `contenteditable` as an I/O device: when input makes its way to the editor, Trix converts that input into an editing operation on its internal document model, then re-renders that document back into the editor. This gives Trix complete control over what happens after every keystroke and avoids the need to use `execCommand` and the inconsistencies that come along with it.
 
-Because these APIs were never fully specified or documented,
-and because WYSIWYG HTML editors are enormous in scope, each
-browser's implementation has its own set of bugs and quirks,
-and JavaScript developers are left to resolve the inconsistencies.
+## Installation
 
-Trix sidesteps these inconsistencies by treating `contenteditable`
-as an I/O device: when input makes its way to the editor, Trix converts that input
-into an editing operation on its internal document model, then re-renders
-that document back into the editor. This gives Trix complete control over what
-happens after every keystroke, and avoids the need to use `execCommand` at all.
+To install Action Text and start working with rich text content, run:
 
-## Installation
+```bash
+$ bin/rails action_text:install
+```
 
-Run `bin/rails action_text:install` to add the Yarn package and copy over the necessary migration. Also, you need to set up Active Storage for embedded images and other attachments. Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+It will do the following:
 
-NOTE: Action Text uses polymorphic relationships with the `action_text_rich_texts` table so that it can be shared with all models that have rich text attributes. If your models with Action Text content use UUID values for identifiers, all models that use Action Text attributes will need to use UUID values for their unique identifiers. The generated migration for Action Text will also need to be updated to specify `type: :uuid` for the `:record` `references` line.
+- Installs the Yarn packages for `trix` and `@rails/actiontext` and adds them to the `application.js`.
```

Comment:
I think it's just better to say JavaScript packages here. The default is Importmap, indeed, and in that case you won't use Yarn. You could use Bun or NPM too

Reply:
Right, not yarn anymore.